### PR TITLE
allow unqualified .svc domain

### PR DESCRIPTION
**Pull request by rajatjindal**

### CODE REVIEW

**Comment by rajatjindal** [comment]
once kapprover is merged, will need to fix dependencies source

**Comment by rajatjindal** [comment]
this is ready for review as kapprover changes have been merged/released.

File: main.go [hunk 1]
```diff
@@ -50,6 +51,7 @@ func main() {
 	flag.StringVar(&additionalIps, "ips", "", "additional IP addresses that resolve to this Pod; comma separated")
 	flag.StringVar(&additionalDnsNames, "dns-names", "", "additional dns names; comma separated")
 	flag.StringVar(&serviceNames, "service-names", "", "additional service names that resolve to this Pod; comma separated")
+	flag.BoolVar(&allowUnqualified, "allow-unqualified", false, "allow unqualified .svc domain")
```
Comment (johngmyers) [inline main.go]: `include-unqualified` ?

Comment (johngmyers) [reply main.go]: or `request-unqualified`?

**Review by johngmyers — APPROVED** [review]

**Pull request merged**